Frontend/Gmfctn/src/app/modules/+dashboard/components/top-chart/top-chart.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { SayThankModalComponent } from '../../../../shared/components/say-thank-modal/say-thank-modal.component';
+import { Roles, User } from 'src/app/shared/models/user';
+import { Graph } from 'src/app/shared/models/graph';
+
+@Component({
+  selector: 'app-top-chart',
+  templateUrl: './top-chart.component.html',
+  styleUrls: ['./top-chart.component.scss']
+})
+export class TopChartComponent implements OnInit {
+  users: User[] = [{
+                    firstName: 'Petro',
```

Comment:
The same here, opening and a closing of an object on a new lines

Reply:
Fixed